addons/website_sale_picking/models/res_config_settings.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    picking_site_ids = fields.Many2many(
+        'delivery.carrier',
+        related='website_id.picking_site_ids',
+        readonly=False
+    )
+
+    picking_product_id_default = fields.Many2one('product.product',
+                                                 store=False,
+                                                 default=lambda self: self.env.ref('website_sale_picking.onsite_delivery_product'),
+                                                 readonly=True)
```

Comment:
I think this field isn't useful but it allows me to help you a bit regarding some details :)

```suggestion
    default_picking_product_id = fields.Many2one(
        'product.product', store=False, readonly=True,
        default=lambda self: self.env.ref('website_sale_picking.onsite_delivery_product'))
```

What we can say is that related field names are, by convention, suffixed with `_id(s){0,1}` (s is added for X2many) and regarding the args, we can return to the line to have something clearer, indented one time. One can argue that you have to declare one arg per line, it's ok for me too. Here I find it clearer like I suggest.

I'll give you the reason why we may avoid that kind of field readonly, not stored not computed with a constant value.